app/src/main/java/com/framgia/chat_03/screen/home/HomeActivity.java

```diff
@@ -121,4 +127,31 @@ public void showAvatar(String image) {
                 .apply(RequestOptions.circleCropTransform())
                 .into(mImageAvatar);
     }
+
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.image_logout:
+                new AlertDialog.Builder(this)
+                        .setTitle(getResources().getString(R.string.log_out))
+                        .setMessage(getResources().getString(R.string.log_out_mes))
+                        .setPositiveButton(getResources().getString(R.string.log_out_yes),
+                                new DialogInterface.OnClickListener() {
+                                    @Override
+                                    public void onClick(DialogInterface dialog, int which) {
+                                        startActivity(SignInActivity.getIntent(HomeActivity.this));
+                                        dialog.dismiss();
+                                    }
+                                })
+                        .setNegativeButton(getResources().getString(R.string.log_out_no),
```

Comment:
nếu k xử lý gì thì có thể để null action 
```suggestion
                        .setNegativeButton(getResources().getString(R.string.log_out_no), null)
```